### PR TITLE
Create alpha.js

### PR DESCRIPTION
**Pull request by priyanshuuranjan**

### CODE REVIEW

File: alpha.js [hunk 1]
```diff
@@ -0,0 +1 @@
+HHYYYY
```
Comment (riddhiiiiiiiii) [inline alpha.js:1]: done

**Review by riddhiiiiiiiii — COMMENTED** [review]
done

**Review by priyanshuuranjan — COMMENTED** [review]
grvrs

**Review by riddhiiiiiiiii — APPROVED** [review]
finnee

**Comment by priyanshuuranjan** [comment]
gdfdsh

**Pull request merged**